content/pages/support.mdx

```diff
@@ -5,13 +5,4 @@ export const meta = {
     heading: 'Commercial Support',
     description: 'Need a little more help?'
   },
-  intro: {
-    heading: 'Venafi',
-    description:
-      'Venafi is the principal maintainer of the cert-manager project and works directly with the CNCF. Commercial support for cert-manager is available as part of Venafi\'s Long Term Support (LTS) offering which includes access to expertise to help organisations scale effectively and securely with cert-manager.',
-    cta: {
-      logo: '/images/venafi-lts-support.png',
```

Comment:
suggestion: It looks like this image isn't used anywhere else and we're removing it here - could we remove it? It's at `public/images/venafi-lts-support.png`

Reply:
### You are correct. Removed in the latest push